src/status_im/ui/screens/discover/subs.cljs

```diff
@@ -49,3 +49,12 @@
           current-tag (:current-tag db)
           tags        (:discover-search-tags db)]
       (get-discoveries-by-tags discoveries current-tag tags))))
+
+(reg-sub :get-all-dapps
+  (fn [db]
+    (let [dapp? (->> (get-in db [:group/contact-groups "dapps" :contacts])
+                     (map :identity)
+                     set)]
+      (->> (:contacts/contacts db)
+           (filter (fn [[k _]] (dapp? k)))
```

Comment:
it could be `(filter #(-> % key dapp?))`, though not a big deal. `comp` can be used but it reverts order, harder to read.

Reply:
True, will change it to this